model/parsley/parsley.go

```diff
@@ -0,0 +1,43 @@
+package parsley
+
+import (
+	"regexp"
+
+	"github.com/mongodb/grip"
+)
+
+// ParsleyFilter represents a filter for the Parsley log viewer. Parsley filters can be defined at
+// the project-level and at the user-level.
+type ParsleyFilter struct {
```

Comment:
Since this is within a package called Parsley already, it's good practice to just call this struct "Filter" , to reduce naming duplication (`parsley.Filter` as opposed to `parsley.ParsleyFilter`) 
https://google.github.io/styleguide/go/decisions#package-vs-exported-symbol-name